source/_integrations/bluetooth_le_tracker.markdown

```diff
@@ -58,27 +58,31 @@ Enabling the battery tracking might slightly decrease the duration of the batter
 
 For additional configuration variables check the [Device tracker page](/integrations/device_tracker/).
 
-## Home Assistant Core installs
+{% details %}
```

Comment:
This details block is now used once, just the blue page.

I wonder if we should change the syntax before using it more widely.

The problem is that YAML syntax in these block often causes problems with formatters in editors like VSCode.

We can maybe simplify the use by using the default syntax of Liquid for this.

e.g.,

```
{% details "Title..." %}

Regular content

{% details %}
```

Reply:
It was designed to have multiple "items" inside each block.
But I guess just using multiple blocks works just the same :+1: 
I'll rewrite that plugin